src/providers/irc/AbstractIrcServer.hpp

```diff
@@ -62,7 +62,7 @@ class AbstractIrcServer : public QObject
         const QString &channelName) = 0;
 
     virtual void privateMessageReceived(Communi::IrcPrivateMessage *message);
-    virtual void readConnectionMessageReceived(Communi::IrcMessage *message);
+    virtual void readConnectionMessageReceived(Communi::IrcMessage *);
```

Comment:
why is this changed in the header?

Reply:
The source warning was:

`warning: unused parameter 'message'`

After removing it there, I thought I saw a warning about the header. I don't see that now. Maybe I'm missing something.